### PR TITLE
Test ashr instructions with the `exact` keyword.

### PR DESCRIPTION
**Pull request by ptersilie**
Requires https://github.com/ykjit/ykllvm/pull/161

### CODE REVIEW

File: tests/ir_lowering/unsupported_variants.ll [hunk 1]
```diff
@@ -12,7 +12,6 @@
 ;         br bb2
 ;      bb2:
 ;         unimplemented <<  %{{13}} = fadd nnan float %{{3}}, %{{3}}>>
-;         unimplemented <<  %{{14}} = udiv exact i32 %{{2}}, 1>>
```
Comment (vext01) [inline tests/ir_lowering/unsupported_variants.ll:15]: How comes we removed this?

Comment (ptersilie) [reply tests/ir_lowering/unsupported_variants.ll:15]: Because we now ignore `exact` for all binary operations.

Comment (vext01) [reply tests/ir_lowering/unsupported_variants.ll:15]: ah, no i get it. we are safe to ignore all exacts, not just for ashr.

Comment (vext01) [reply tests/ir_lowering/unsupported_variants.ll:15]: Sorry, we did discuss this, just it slipped the mind :)

**Comment by vext01** [comment]
LGTM!

**Pull request merged**